src/layout/encode.ts

```diff
@@ -66,7 +67,8 @@ export default async function(type: Event): Promise<void> {
                 }
                 tokens = tokenize(tokens, metadata);
             }
-            queue(tokens);
+
+            queue(tokens, config.lean ? Event.Backup : type);
```

Comment:
What is this doing and what is Event.Backup?

Reply:
Added comments to clarify. I also explained Event.Backup in a previous comment; it's to back up layout events in memory.

---
In reply to: [368677998](https://github.com/microsoft/clarity-js/pull/166#discussion_r368677998) [](ancestors = 368677998)